doc/specification.md

```diff
@@ -23,16 +23,16 @@ with the mandatory elements object_name, object_type and file_extension.
 Namespaces are converted into brackets, for instance `/NMSPC/CL_OBJECT` becomes `(nmspc)cl_object`.
 
 ### Object Name and Type
-The object name and type represent the ABAP object. Basically, the object name (R3TR) and type are stored in the ABAP object directory (aka TADIR).
+The object name (R3TR) and object type are derived from the object itself, as stored in the ABAP object directory (aka TADIR).
 
 ### Sub Object Name and Sub Object Type
 
-Some object types support sub objects (LIMU object types) like function modules in a function group. For sub objects, the main object name and type must be part of the file name.
+Some objects support sub objects (LIMU object) like function modules in a function group. For sub objects, the main object name and type must be part of the file name.
```

Comment:
I am not sure whether we should stick to `object types` here.

Reply:
Reverted to the previous version.